trailblazer/services/job_service/job_service.py

```diff
@@ -46,7 +53,7 @@ def update_jobs(self, analysis_id: int) -> None:
             if analysis.workflow_manager == WorkflowManager.SLURM:
                 self._update_slurm_jobs(analysis_id)
             if analysis.workflow_manager == WorkflowManager.TOWER:
-                self.store.update_tower_run_status(analysis_id)
+                self.tower_service.update_jobs(analysis_id)
```

Comment:
just my ignorance here, but why the syntax is different for slurm and tower? meaning, why _update_slurm_jobs (starting with _) but update_jobs for tower (without)?

Reply:
Yes, good point! I'll make sure to refactor it to look the same for slurm but I don't want to do more stuff in this PR 😅